ci/rebuild-docker-images.py

```diff
@@ -485,6 +488,98 @@ def playwright_build(distro, arch):
 
 
 
+def appium_build(distro, arch):
+    tag = f"{registry_base}{distro[0]}-{distro[1]}"
+    build_tag(tag, arch, f"""
+FROM openjdk:18-ea-11-jdk-slim-bullseye
+
+ENV HOME=/root \
+    LANG=en_US.UTF-8 \
+    DEBIAN_FRONTEN=noninteractive \
+    LANGUAGE=en_US.UTF-8 \
+    LC_ALL=C.UTF-8 \
+    DISPLAY=:0.0 \
+    DISPLAY_WIDTH=1920 \
+    DISPLAY_HEIGHT=900
+
+
+WORKDIR /
+
+SHELL ["/bin/bash", "-c"]
+
```

Comment:
It can be useful to throw this in here to avoid generating man pages during package installs/upgrades (which is kind of slow, and completely useless for a build container):
```
RUN /bin/bash -c 'echo "man-db man-db/auto-update boolean false" | debconf-set-selections'
```